larq/metrics.py

```diff
@@ -62,7 +63,54 @@ def get_training_metrics():
     return _GLOBAL_TRAINING_METRICS
 
 
-class FlipRatio(Metric):
+class _CompatibilityMetric(Metric):
+    r"""Metric with support for both 1.13 and 1.14+
+    """
+
+    def get_config(self):
+        """Returns the serializable config of the metric."""
+        return {**super().get_config(), "name": self.name, "dtype": self.dtype}
+
+    def reset_states(self):
+        return K.batch_set_value([(v, 0) for v in self.variables])
```

Comment:
Since we only use it here, I'd prefer:
```suggestion
        return tf.keras.backend.batch_set_value([(v, 0) for v in self.variables])
```
But that's just personal preference.